docs/docs/dev/policies/service-health-probes.md

```diff
@@ -22,32 +29,49 @@ networking:
         kuma.io/protocol: http
 ```
 
-This `health.ready` status is intended to show the status of the service itself. It is set differently depending on 
-the environment ([Kubernetes](#kubernetes-probes) or [Universal](#universal-probes)), but it's treated the same way 
-regardless of the environment:
+This `health.ready` status is intended to show the status of the endpoint itself.
+It is set differently depending on the environment ([Kubernetes](#kubernetes) or [Universal](#universal-probes)).
+It's treated the same way regardless of the environment:
 
 - if `health.ready` is true or `health` section is missing - Kuma considers the inbound as healthy and includes it 
   into load balancing set.
-- if `health.ready` is false -  Kuma doesn't include the inbound into load balancing set.
+- if `health.ready` is false -  Kuma doesn't include the inbound into the load balancing set.
 
 Also, `health.ready` is used to compute the status of the Dataplanes and Service. You can see these statuses both in Kuma GUI and Kuma CLI:
 
-- if proxy status is `Offline`, then Dataplane is `Offline`:
+Dataplane health follows these rules:
```

Comment:
IMO either `Dataplane` or just data plane proxy (there are more instances in the rest of the PR)